host-bmc/host_pdr_handler.cpp

```diff
@@ -313,12 +313,14 @@ void HostPDRHandler::fetchPDR(PDRRecordHandles&& recordHandles)
     // main event loop). That way, we can respond to the platform event msg from
     // the host firmware.
     pdrFetchEvent = std::make_unique<sdeventplus::source::Defer>(
-        event, std::bind(std::mem_fn(&HostPDRHandler::_fetchPDR), this,
+        event, std::bind(std::mem_fn(&HostPDRHandler::_fetchPDR), this, tid,
                          std::placeholders::_1));
 }
 
-void HostPDRHandler::_fetchPDR(sdeventplus::source::EventBase& /*source*/)
+void HostPDRHandler::_fetchPDR(uint8_t tid,
+                               sdeventplus::source::EventBase& /*source*/)
 {
+    terminusID = tid;
```

Comment:
if we are setting variable of HostPDRHandler class..why can't we just do it in the ferchPDR function ? why pass it to _fetchPDR and set it here ?

Reply:
done